Default Namespace labels (#26995)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates.md

  the volume mounter.
- `MountPropagation`: Enable sharing volume mounted by one container to other containers or pods.
  For more details, please see [mount propagation](/docs/concepts/storage/volumes/#mount-propagation).

> - `NamespaceDefaultLabelName`: Enable the behavior of APIServer creating a default label `kubernetes.io/metadata.name` containing the namespace name.

Reviewer: ```suggestion
- `NamespaceDefaultLabelName`: Enable the APIServer to create a default label `kubernetes.io/metadata.name` containing the namespace name.
```

Reviewer 2: We call it “the API server” or “kube-apiserver”.

We should mention _what_ gets labelled (ie, namespaces only, not every object in the namespace).

Author: +1

@tengqm if you change your suggestion here, I can commit directly :)

Author: Added here @sftim
